src/app/shared/database/pouch.service.ts

```diff
@@ -0,0 +1,65 @@
+import { Injectable } from '@angular/core';
+import PouchDB from 'pouchdb';
+import PouchDBFind from 'pouchdb-find';
+import { throwError, from } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+import { environment } from '../../../environments/environment';
+
+PouchDB.plugin(PouchDBFind);
+
+@Injectable()
+export class PouchService {
+  private baseUrl = environment.couchAddress;
+  private localDB;
+  private remoteDBs = [];
+
+  constructor() {
+    this.localDB = new PouchDB('local-pouchdb');
```

Comment:
I wonder if this approach of creating one database for PouchDB and storing all dbs from Couch here will cause similar problems that we saw with BeLL-apps of id conflicts.  @dogi wondering what you think?

Reply:
We could create one PouchDB instance per remote database if that's an issue.